### PR TITLE
Kc add delete user servlet

### PR DESCRIPTION
**Pull request by kev1n80**
+ Sort the user's createdListings in descending order based on their timestamp.
  - The most recently created listing will show up first in the user's page.
+ Abstract the function from the delete listing servlet into several functions and move it into a new file.
  - This allows the delete user servlet to leverage these functions to delete the users created listings.
+ Add a delete user functionality.
+ Add a delete button to the user page.
  - When clicked will call the delete user servlet and log out the user.
Screenshot 2020-08-28 at 3 11 16 PM
Screenshot 2020-08-28 at 3 12 53 PM

### CODE REVIEW

File: src/main/webapp/scripts/user.js [hunk 1]
```diff
@@ -105,9 +108,22 @@ function createUserInformation(bio, email, name) {
       createAElement('Create listing', 'newlisting.html', '', 'card-button', '')
       );
 
+  divCardInformation.appendChild(
+    createDeleteElements('user-delete', logoutUser, email, '/delete-user'));
```
Comment (zrideaux) [inline src/main/webapp/scripts/user.js:112]: Add an extra 2 spaces to line 112

File: src/main/webapp/stylesheets/master.css [hunk 2]
```diff
@@ -85,6 +85,7 @@ header {
 /* Used to style delete related material-icons */
 .material-icons.delete-icon:hover {
   color: red;
+  cursor: pointer;
```
Comment (zrideaux) [inline src/main/webapp/stylesheets/master.css:88]: I didn't know about this! Cool usage!

File: src/main/webapp/scripts/user.js (hunk 1 above)
Comment (kev1n80) [reply src/main/webapp/scripts/user.js]: Nice catch!

File: src/main/webapp/stylesheets/master.css (hunk 2 above)
Comment (kev1n80) [reply src/main/webapp/stylesheets/master.css:88]: Thanks!

**Pull request merged**